frank-doc-doclet/src/test/java/org/frankframework/frankdoc/wrapper/FrankClass2Test.java

```diff
@@ -126,8 +126,8 @@ public void getDeclaredMethodsDoesNotIncludeInheritedMethods() throws FrankDocEx
 		final Set<String> actualMethodNames = new TreeSet<>();
 		Arrays.asList(declaredMethods).forEach(m -> actualMethodNames.add(m.getName()));
 		List<String> sortedActualMethodNames = new ArrayList<>(actualMethodNames);
-		sortedActualMethodNames = sortedActualMethodNames.stream().filter(name -> !name.contains("jacoco")).collect(Collectors.toList());
-		assertArrayEquals(new String[]{"getMyInnerEnum", "getProtectedStuff", "methodWithoutAnnotations", "myAnnotatedMethod", "setInherited", "setVarargMethod"}, sortedActualMethodNames.toArray());
+		sortedActualMethodNames = sortedActualMethodNames.stream().filter(name -> !name.contains("jacoco")).toList();
```

Comment:
Misschien zeg ik nu iets wat je allang weet: .toList() resulteert in een immutable list (wat hier niet erg is), maar wat in andere situaties dus wel problemen kan geven. Collectors.toList is mutable.

Reply:
Ja klopt. Goed om even bij stil te staan. Ik wil het alleen veranderen als het immutable mag zijn.